server/src/main/java/org/elasticsearch/search/fetch/subphase/FetchSourceContext.java

```diff
@@ -225,6 +344,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.startObject();
             builder.array(INCLUDES_FIELD.getPreferredName(), includes);
             builder.array(EXCLUDES_FIELD.getPreferredName(), excludes);
+            builder.field(INCLUDE_VECTORS.getPreferredName(), includeVectors);
```

Comment:
`include_vectors` should be written to XContent only when it's not set to the default value